kerchunk/intro.qmd

```diff
@@ -0,0 +1,40 @@
+---
+title: "Kerchunk"
+subtitle: References for Chunked (maybe), Compressed (maybe) N-Dimensional Arrays
+---
+
+# What is Kerchunk?
+
+Kerchunk's potential utility to the geospatial data community cannot be overstated. Kerchunk enables cloud-optimized access to traditional geospatial file formats, like NetCDF, potentially negating the need to create and store copies of data for cloud-optimized access.
+
+Kerchunk creates Zarr metadata, stored in a "reference file", for non-Zarr data. Kerchunk enables a unified way to access chunked, compressed n-dimensionsional data across a variety of conventional data formats. The Kerchunk library now supports NetCDF/HDF5, GRIB2, TIFF.
+
+Learn more about Kerchunk at [kerchunk.readthedocs.io](https://kerchunk.readthedocs.io/en/latest/).
+
+# How to Kerchunk 
+
+Kerchunk is a `ReferenceFileSystem` generator. The powerful `fsspec` library provides a uniform file system interface to many different storage backends and protocols. Kerchunk generates data the `fsspec.ReferenceFileSystem` can use to read data.
```

Comment:
I think I'd swap these sentences to first introduce fsspec and then explain how kerchunk fits into that. Maybe

```suggestion
The powerful [`fsspec` library](https://filesystem-spec.readthedocs.io/en/latest/) provides a uniform file system interface to many different storage backends and protocols. In addition to abstracting over existing protocols, its [`ReferenceFileSystem`](https://filesystem-spec.readthedocs.io/en/latest/api.html#fsspec.implementations.reference.ReferenceFileSystem) class lets you view byte ranges of some other file as a file system. Kerchunk helps in easily generating these `ReferenceFileSystem` objects.
```

Thoughts?